### PR TITLE
Scp 2704 markdown

### PR DESCRIPTION
**Pull request by jhbertra**
JIRA: https://jira.iohk.io/browse/SCP-2704

Fixes contract cards rendering raw markdown in the description. Also cleans up a few warnings and adds purescript-language-server to the nix shell.

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -16,6 +16,7 @@ This includes a variety of useful tools:
 * Agda with the right libraries set up to build our Agda code
 * `stylish-haskell`
 * `haskell-language-server`
+* `purescript-language-server`
```
Comment (hrajchert) [inline CONTRIBUTING.adoc:19]: I was able to use the language-server + IDE without having to add it to nix-shell 🤔 ... But I don't think this hurts either way.

The only caveat is that you need to open as the root folder each purescript project, which is mitigated by the fact that you can have multiple projects in the same workspace

**Review by hrajchert — APPROVED** [review]
LGTM

File: CONTRIBUTING.adoc (hunk 1 above)
Comment (jhbertra) [reply CONTRIBUTING.adoc:19]: What editor do you use? I think the VS Code plugin automatically downloads and installs purescript-language-server for you, so you might have it installed in your global environment

Comment (hrajchert) [reply CONTRIBUTING.adoc:19]: ah, I do use VS Code, maybe later we can clarify this in the docs, if you use vscode is not necessary to launch the language-server manually

**Pull request merged**